### ISSUE TITLE
Adding aria/alt tag support for images

### ISSUE THREAD

**Issue by frankie-tech** [biati-digital/glightbox#118]
**Please describe the feature you want to be implemented.**
Please note: The features should be something that everyone can use and not just something you need for your project. 

I recently had to do an accessibility audit for a client and found that the images that were generated had no alt tags associated with them. I double checked the docs, and realized it was a feature I had made up. Still, I think it is one that can be useful. Increasing accessibility is a win for everybody.

**Explain why the feature is useful**
For the same reasons that it is useful to handle proper focus, a proper alt/aria-label/labeledby attribute makes GLightbox the most accessible it can be. One example would be a light box that demonstrates a series of instructions.

**Have you seen it somewhere else?**
I don't have any places where I've seen other implementations, but I do have some examples that can help in creating a solution.

[David Walsh - Empty alt attributes](https://davidwalsh.name/accessibility-tip-empty-alt-attributes)

[MDN - aria-describedby example](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute#Example_2_A_Close_Button)

**Additional context**
I'm afraid I don't have any additional context, but I have worked up a simple implementation on a fork I made [here](https://github.com/frankie-tech/glightbox/blob/add-accessibility-features/src/js/glightbox.js#L733+L739)

If there're changes I need to make lemme know, I'd be happy to keep working on them 😁

**Comment by biati-digital** [comment]
hi, you are right about the alt tag, I think your changes will work without problems, can you send a pull request?

**Comment by frankie-tech** [comment]
🎉 PR made! 🎉
